firebase/podcasts.js

```diff
@@ -3,9 +3,17 @@ import firebase from 'firebase/app'
 /**
  * Fetch podcasts
  */
+
+var collection = 'podcasts'
```

Comment:
Prefer using "const" instead of "var" (I suggest you to configure eslint in your IDE from the local config to be notified in real time of js best practices ;) )